storage-api/src/java/org/apache/hive/common/util/SuppressFBWarnings.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.hive.common.util;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.CLASS)
+public @interface SuppressFBWarnings {
```

Comment:
Isn't there a SuppressFBWarnings annotation in findbugs package? Why do we need to introduce our own annotation?

Reply:
Hey @mustafaiman  -- in the main pom.xml we are only using the finbugs-maven-plugin and not the findbugs-annotation package (only hive-ql is using it as dependency and I would like to remove it eventually).

In order to avoid adding jsut another dependency (findbugs-annotation), I implemented the annotation as a separate class, Findbugs doesn't care in which package the annotation is, so it works pretty well -- ideally we can also reuse it across hive packages.

Reply:
If we are defining our own annotation, I would like to put it in a common to prevent each module having their own copy of the same annotation. I thought hive-common would be a good candidate but storage-api does not depend on hive-common either. This looks like the only way then. Thanks for the link.